lib/charms/layer/nagios.py

```diff
@@ -0,0 +1,53 @@
+import os
+import stat
+
+NAGIOS_PLUGINS_DIR = '/usr/lib/nagios/plugins'
+
+
+def install_nagios_plugin(source_path, plugin_name, data=None):
+    """ Install a nagios plugin.
+
+    Args:
+        source_path: Source file path to install
+        plugin_name: Name of the plugin in nagios
+
```

Comment:
Please include `data` in the Args section - it's not clear what it is. It overrides the contents of `source_path`?